mllib-local/src/test/scala/org/apache/spark/ml/util/TestingUtils.scala

```diff
@@ -31,7 +31,7 @@ object TestingUtils {
    * Note that if x or y is extremely close to zero, i.e., smaller than Double.MinPositiveValue,
    * the relative tolerance is meaningless, so the exception will be raised to warn users.
    */
-  private def RelativeErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
+  private[ml] def RelativeErrorComparison(x: Double, y: Double, eps: Double): Boolean = {
```

Comment:
might it be better to just make this public, if we are using it in tests, similar to other test methods?